client/layout/guided-tours/tours/simple-payments-email-tour/index.js

```diff
@@ -19,13 +18,9 @@ import {
 	Quit,
 } from 'layout/guided-tours/config-elements';
 import { AddContentButton } from 'layout/guided-tours/button-labels';
-import { getSectionName, hasSidebar } from 'state/ui/selectors';
 import { targetForSlug } from 'layout/guided-tours/positioning';
 import { localizeUrl } from 'lib/i18n-utils';
 
-const sectionHasSidebar = ( state ) =>
-	hasSidebar( state ) && ! includes( [ 'customize' ], getSectionName( state ) );
```

Comment:
This guided tour isn't finished yet, see the todo section in the PR description. I'll need to rewrite the `when` condition to use some DOM observer that waits for the sidebar element to appear.

Reply:
Gotcha 👍

Reply:
@tyxla After I started looking at this guided tour, I realized it can be removed completely 🙂 See #45286 for all the details. After that removal PR is merged, this one can be rebased and will be ready for review and deploy.

Reply:
Sounds great! I've reviewed that PR to unblock this one.